### PR TITLE
Lps 118086 part 3 alternative 1

### PR DESCRIPTION
**Pull request by arthurchan35**

### CODE REVIEW

File: modules/apps/portal-crypto-hash-provider/portal-crypto-hash-provider-message-digest/src/main/java/com/liferay/portal/crypto/hash/provider/message/digest/internal/MessageDigestCryptoHashProvider.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash-provider/portal-crypto-hash-provider-message-digest/src/main/java/com/liferay/portal/crypto/hash/provider/message/digest/internal/MessageDigestCryptoHashProvider.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/portal-crypto-hash-provider/portal-crypto-hash-provider-message-digest/src/main/java/com/liferay/portal/crypto/hash/provider/message/digest/internal/factory/MessageDigestCryptoHashProviderFactory.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash-provider/portal-crypto-hash-provider-message-digest/src/main/java/com/liferay/portal/crypto/hash/provider/message/digest/internal/factory/MessageDigestCryptoHashProviderFactory.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/facet/NestedFacetProcessor.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/facet/NestedFacetProcessor.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/NestedFacetImpl.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/NestedFacetImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/nested/NestedFacetFactoryImpl.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/nested/NestedFacetFactoryImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/nested/NestedFacetSearchContributorImpl.java [hunk 6]
```diff
@@ -0,0 +1,178 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/nested/NestedFacetSearchContributorImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/provider/JournalArticleAnalyticsReportsInfoItemObjectProvider.java [hunk 7]
```diff
@@ -0,0 +1,53 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/provider/JournalArticleAnalyticsReportsInfoItemObjectProvider.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:sf

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by csierra** [comment]
resending a smaller one here: https://github.com/liferay-appsec/liferay-portal/pull/269

**Pull request closed without merging**